### PR TITLE
Adds hot ice

### PR DESCRIPTION
**Pull request by LemonInTheDark**
## About The Pull Request

Adds hot ice coffee, and hot ice slush. Ya grind some hot ice into slush, it can melt into enough plasma to match the og grind requirements, or mixed 1 to 2 with coffee to make some extra cold joe
## Why It's Good For The Game

Uses for hot ice????!@?!??! Tetra conviced me to do this. Still not sure how. Copy of [TG Pr](https://github.com/tgstation/tgstation/pull/52208)
## Changelog
🆑 LemonInTheDark, TetraZeta for the sprites
add: Adds hot ice slush, grind some hot ice to make it, it'll melt down to plasma if you heat it to 30c.
add: Adds hot ice coffee, mix 1 part hot ice slush with 2 parts coffee to make some. Careful it's kinda cold.
/🆑

### CODE REVIEW

File: code/modules/reagents/chemistry/recipes/toxins.dm [hunk 1]
```diff
@@ -40,9 +40,10 @@
 	results = list(/datum/reagent/toxin/mutagen = 3)
 	required_reagents = list(/datum/reagent/uranium/radium = 1, /datum/reagent/phosphorus = 1, /datum/reagent/chlorine = 1)
 
-/datum/chemical_reaction/lexorin
-	results = list(/datum/reagent/toxin/lexorin = 3)
-	required_reagents = list(/datum/reagent/toxin/plasma = 1, /datum/reagent/hydrogen = 1, /datum/reagent/medicine/salbutamol = 1)
```
Comment (triplezeta) [inline code/modules/reagents/chemistry/recipes/toxins.dm:45]: is there a reason for this removal?

**Comment by LemonInTheDark** [comment]
Dear mark, if you merge this squash it, or you're gonna have a bad time

File: code/modules/reagents/chemistry/recipes/toxins.dm (hunk 1 above)
Comment (LemonInTheDark) [reply code/modules/reagents/chemistry/recipes/toxins.dm:45]: woop

Comment (triplezeta) [reply code/modules/reagents/chemistry/recipes/toxins.dm:45]: this is on your original PR as well, h

**Comment by MarkSuckerberg** [comment]
image
You underestimate my hatred of those who don't squash

File: code/modules/reagents/chemistry/recipes/toxins.dm (hunk 1 above)
Comment (LemonInTheDark) [reply code/modules/reagents/chemistry/recipes/toxins.dm]: you saw nothing 
image

**Pull request merged**